content/en/docs/ops/best-practices/security/index.md

```diff
@@ -31,7 +31,9 @@ It takes effort to configure the correct authorization policies to best protect
 It is important to understand the implications of these configurations as Istio cannot determine the proper authorization for all users.
 Please follow this section in its entirety.
 
-### Apply default-deny authorization policies
+### Safer Authorization Policy Pattern
+
+#### Apply default-deny authorization policies
```

Comment:
And we seem to go between pattern and policy. Not sure if it makes sense to change policies here to pattern.

Reply:
yes good point, updated the title to `use default-deny pattern` be more consistent.